Frontend/src/api/urls.js

```diff
@@ -1 +1,2 @@
-export const LOCAL_URL = 'http://localhost:5000/'
+// export const LOCAL_URL = 'http://35.197.238.123/api'
+export const LOCAL_URL = 'http://127.0.0.1:8000'
```

Comment:
@Aibaki The line `export const LOCAL_URL = 'http://127.0.0.1:8000/', is only applicable to the docker specific deployment method and won't work for the Ubuntu deployment.

Please identify it in the deployment doc as a mandatory change pre-deployment and consider making it non-specifi, eg. `export const LOCAL_URL = 'http://<api-address>:<api-port>/',`